### PR TITLE
Darwinia => Ethereum

### PR DESCRIPTION
**Pull request by wuminzhe**
- [x] The authority monitors the `NewMMRRoot` event. Then the authority signs `keccak256((spec_name, block_for_mmr_root, mmr_root))` and send the signature to darwinia
- [x] The normal relayer monitors the `AuthoritiesSetSigned` event. Then send the data to ethereum.
- [x] Global extrinsics sending queue

### CODE REVIEW

**Comment by HackFisher** [comment]
Related: https://github.com/tomusdrw/rust-web3/issues/436

**Review by HackFisher — APPROVED** [review]
LGTM. @wuminzhe Is this PR ready for merging?

**Comment by wuminzhe** [comment]
I will fix [104](https://github.com/darwinia-network/bridger/issues/104) first

**Comment by wuminzhe** [comment]
@HackFisher @AurevoirXavier This PR can be reviewed. 

It seems not good enough to prevent resending of tx after listening to an old event. But if I want to optimize it, it may need to modify the chain or add local storage to remember the processed events.

**Review by HackFisher — APPROVED** [review]
@wuminzhe I'm not sure the issue of decoding MMRRootsToSign storage(None/Some([])) resolved or not, the left LGTM for now.

**Comment by HackFisher** [comment]
@wuminzhe CI failed.

File: Cargo.toml [hunk 1]
```diff
@@ -70,9 +74,11 @@ features = ["derive"]
 # jsonrpsee = { path = "../jsonrpsee", features = ["ws"] }
 # substrate-subxt = { path = "../substrate-subxt" }
 jsonrpsee = { git = "https://github.com/AurevoirXavier/jsonrpsee", branch = "xavier-send-error", features = ["ws"] }
-substrate-subxt = { git = "https://github.com/AurevoirXavier/substrate-subxt", branch = "xavier-send-error" }
+substrate-subxt = { git = "https://github.com/wuminzhe/substrate-subxt", branch = "xavier-send-error" }
```
Comment (HackFisher) [inline Cargo.toml:77]: Changing deps.

Comment (wuminzhe) [reply Cargo.toml:77]: I'll check it.

**Comment by HackFisher** [comment]
@wuminzhe Ready for merge

**Comment by wuminzhe** [comment]
@HackFisher I have finished my work, you can merge now

**Pull request merged**